css/evaluations.css

```diff
@@ -0,0 +1,50 @@
+body{
+	background-color: white;
+}
+
+.footer{
+	margin-bottom: 0;
+}
+
+.mainTableClass{
+	width: 60%;
+	margin: 0 auto;
+	margin-top: 100px;
+	margin-bottom: 100px;
+
+	font-size: 20px;
+	border-collapse: collapse;
+}
+
+.mainTableClass th{
+	text-align: left;
+}
+
+.mainTableClass tr{
+	height: 50px;
+
+	border-bottom: 1px solid black;
+}
+
+.mainTableClass th,td{
```

Comment:
The new table has the same padding like the old one. I didn't expected something else. Maybe I don't get what you want to say.